src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -75,6 +82,7 @@ class Snapper:
     """
 
     def __init__(self):
+
```

Comment:
We don't need a new line here because we don't have docstring. The docstring was already placed in the class definition above.